template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/Company.ClassLibrary1.csproj

```diff
@@ -5,7 +5,8 @@
     <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.ClassLibrary1</RootNamespace>
     <LangVersion Condition="'$(langVersion)' != ''">$(ProjectLanguageVersion)</LangVersion>
-    <Nullable Condition="('$(Framework)' != 'netstandard2.0')">enable</Nullable>
+    <DisableImplicitNamespaceImports Condition="'$(csharpFeature_GlobalUsings_Supported)' != 'true'">true</DisableImplicitNamespaceImports>
```

Comment:
This was the other thing I was going to comment on. If we need this, it's a bug in the feature - if we aren't where implicit usings are supported, they should not need to be disabled. 

However, let's get Preview 7 out working correctly (correctly being you can run templates and they then compile), so if this is needed today, please leave it in.

Reply:
We need it when language version is < 10.0, otherwise it won't compile.